CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+
+## 2.10.0-rc.1
+
+### Grafana Mimir
+
+* [FEATURE] The following features are no longer considered experimental. #5872
+  * Ruler storage cache (`-ruler-storage.cache.*`)
+  * Exclude ingesters running in specific zones (`-ingester.ring.excluded-zones`)
+  * Cardinality-based query sharding (`-query-frontend.query-sharding-target-series-per-shard`)
```

Comment:
I just noticed that this is not marked as "advanced" but I think it should.

Reply:
Adjusted in https://github.com/grafana/mimir/pull/5872/commits/0535131e9e598ac50a4d35a6a7b5368e208a0a60